tutorials/introductory/animation_tutorial.py

```diff
@@ -0,0 +1,249 @@
+    [ax.scatter(x_frames[:, i], y_frames[:, i], c="b")]
+    for i in range(x_frames.shape[-1])
+]
+
+ani = animation.ArtistAnimation(fig=fig, artists=artists, repeat_delay=1000)
+plt.show()
+
+###############################################################################
+# Animation Writers
+# =================
+#
+# Animation objects can be saved to disk using various multimedia writers
+# (ex: Pillow, *ffpmeg*, *imagemagick*). Not all video formats are supported
+# by all writers. There are 4 major types of writers:
+#
+# - :class:`~matplotlib.animation.PillowWriter` - Uses the Pillow library to
+#   create the animation.
+#
+# - :class:`~matplotlib.animation.HTMLWriter` - Used to create JS-based
+#   animations.
+#
+# - Pipe-based writers - :class:`~matplotlib.animation.FFMpegWriter` and
+#   :class:`~matplotlib.animation.ImageMagickWriter` are pipe based writers.
+#   These writers pipe each frame to the utility (*ffmpeg* / *imagemagick*)
+#   which then stitches all of them together to create the animation.
+#
+# - File-based writers - :class:`~matplotlib.animation.FFMpegFileWriter` and
+#   :class:`~matplotlib.animation.ImageMagickFileWriter` are examples of
+#   file-based writers. These writers are slower than their standard writers
+#   but are more useful for debugging as they save each frame in a file before
+#   stitching them together into an animation.
+#
+# ================================================  ===========================
+# Writer                                            Supported Formats
+# ================================================  ===========================
+# :class:`~matplotlib.animation.PillowWriter`       .gif, .apng
+# :class:`~matplotlib.animation.HTMLWriter`         .htm, .html, .png
+# :class:`~matplotlib.animation.FFMpegWriter`       All formats supported by
+#                                                   *ffmpeg*
+# :class:`~matplotlib.animation.ImageMagickWriter`  .gif
+# ================================================  ===========================
+#
+# To save animations using any of the writers, we can use the
+# `.animation.Animation.save` method. It takes the *filename* that we want to
+# save the animation as and the *writer*, which is either a string or a writer
+# object. It also takes an *fps* argument. This argument is different than the
+# *interval* argument that `~.animation.FuncAnimation` or
+# `~.animation.ArtistAnimation` uses. *fps* determines the frame rate that the
+# **saved** animation uses, whereas *interval* determines the frame rate that
+# the **displayed** animation uses.
+
+fig, ax = plt.subplots()
+ax.grid()
+rng = np.random.default_rng()
+
+scat = ax.scatter(
+    rng.uniform(low=0, high=1, size=100),
+    rng.uniform(low=0, high=1, size=100),
+    c="b"
+)
+
+
+def update(frame):
+    x = rng.uniform(low=0, high=1, size=100)
+    y = rng.uniform(low=0, high=1, size=100)
+    data = np.stack([x, y]).T
+    scat.set_offsets(data)
+    return (scat,)
+
```

Comment:
I wasn't saving anything here in particular because it wouldn't make a lot of sense to actually save something in the tutorial while making the docs? Maybe I'm wrong there. The literal blocks look like a good idea, I'll take a look, thanks!